taxonomy-editor-frontend/src/pages/editentry/index.jsx

```diff
@@ -0,0 +1,44 @@
+import { createTheme, ThemeProvider, Typography } from "@mui/material";
+import { useParams } from "react-router-dom";
+import FetchRelations from "./FetchRelations";
+import useFetch from "../../components/useFetch";
+import ListAllEntryProperties from "./ListAllEntryProperties";
+import ListAllOtherProperties from "./ListAllOtherProperties";
+
+const EditEntry = () => {
+    let url = 'http://localhost:80/'
+    let isEntry = false;
+    const { id } = useParams();
+
+    if (id.startsWith('__header__')) { url += 'header/' }
+    else if (id.startsWith('__footer__')) { url += 'footer/' }
+    else if (id.startsWith('synonym')) { url += `synonym/${id}/` }
+    else if (id.startsWith('stopword')) { url += `stopword/${id}/` }
+    else { url += `entry/${id}/`; isEntry = true; }
+
+    const { data: node, error, isPending } = useFetch(url);
+    let nodeObject = null;
+    if (!isPending) {
+        nodeObject = node[0];
+    }
+
+    const theme = createTheme({

```

Comment:
better avoid having partial small custom themes. If possible move it to the top of the tree